packages/client/package.json

```diff
@@ -10,6 +10,7 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
+    "typescript": "^4.4.2",
```

Comment:
Is this actually needed - we aren't using TypeScript (yet?)
If it's fixing a bug let me know.